.github/workflows/publish.yml

```
@@ -25,15 +25,17 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: 3.12
-      - name: Install flit
+      - name: Install dependencies
         run: |
-          pip install flit
-      - name: Publish the distibution to PyPI
-        run: flit publish
-        env:
-          FLIT_INDEX_URL: https://upload.pypi.org/legacy/
-          FLIT_USERNAME: __token__
-          FLIT_PASSWORD: ${{ secrets. PYPI_API_TOKEN }}
+          python -m pip install --upgrade pip
+          pip install build
+      - name: Build package
+        run: python -m build
+      - name: Publish package
+        uses: pypa/gh-action-pypi-publish@2f6f737ca5f74c637829c0f5c3acd0e29ea5e8bf
```

```suggestion
        uses: pypa/gh-action-pypi-publish@v1
```

Should get us fewer dependabot pull requests